src/Dfc.CourseDirectory.Web/Controllers/ApprenticeshipsController.cs

```diff
@@ -102,7 +102,7 @@ public async Task<IActionResult> Search([FromQuery] SearchRequestModel requestMo
 
                 if (result.IsSuccess && result.HasValue)
                 {
-                    foreach (var item in result.Value)
+                    foreach (var item in result.Value.Where(r => r.ApprenticeshipType != ApprenticeshipType.FrameworkCode))
```

Comment:
Wondering out loud if we should whitelist rather than blacklist here. On the plus side it would mean there's no mention here of a concept from the past (Frameworks).